api/routes/sequence.py

```diff
@@ -134,7 +159,7 @@ async def get_latest_sequence_id_from_sample_id_and_type(
     '/all-for-sample-id/{sample_id}',
     operation_id='getAllSequencesForSampleId',
 )
-async def get_all_sequence_id_for_sample_id(
+async def get_all_sequences_id_for_sample_id(
```

Comment:
@violetbrina I don't remember doing this. Was it on purpose? 😅

Reply:
yeah fixing a typo